source/documentation/using_the_api/api_reference.md

```diff
@@ -338,32 +340,22 @@ Example response:
 }
 ```
 
-### <a name="downloadreg"></a>`GET /download-register`
-
-Download the full contents of the register in a ZIP file.
-
-*Note: This will download every entry and item as an individual JSON file. If you only want to download records, use `GET /records`.*
-
-Example URL: `https://local-authority-eng.register.gov.uk/download-register`
-
-Example request: `curl -o localauthorityeng.zip --request GET --url https://local-authority-eng.register.gov.uk/download-register`
-
 ## Getting updates
 
 You can download the new entries in 2 ways:  
 
-* download another copy of the full register
+* download a copy of the full register
 * download individual updated entries for the records you are using
```

Comment:
If I'm being picky "updated entries" is abuse of language given that entries are immutable. Records on the other hand are recomputed for any change in a Register. @michaelabenyohai am I correct?

"Could it be changed to something on the lines of "download new entries for the records you are interested in". (I don't know if I like the "you are using/you are interested in" but besides the point I want to get across.

Reply:
My understanding of the language is the same. "Download new entries for the records you are interested in" sounds like quite a mouthful, but it gets the point across better.

To my mind, the problem is if you don't say something like "...you are interested in" then you're left with something like "...of interest" or "...in question" and out of context, those doesn't make as much sense. So "...you are interested in" is a bit of a mouthful, but it might be the least worst option for now.

Reply:
Agree that "updated entries" is not correct and we should be talking about "new entries" instead. As you say records change (as a function of the new entries in the register) - records are the things that are mutable, not entries.

Reply:
Also - what do we think about using the word "download" in the context of API usage? I'm possibly being a bit picky here and maybe it's more relevant to a different PR. Should we use "get" instead? (Unless we are explicitly talking about the `/download-register` link, where it does make sense as it downloads files.

Reply:
What do you mean by "files"?

Reply:
It downloads a `.zip` file

Reply:
So, being picky, it lets you get the whole register as `application/zip` (and once uncompressed you get a set of JSON files)?

I'm not particularly bothered by the word "download" but I'm fine keeping everything as "get" when you ask for a resource and "download" when you ask for a compressed dump.